```
# go.podman.io/common/pkg/config
vendor/go.podman.io/common/pkg/config/new.go:58:58: syntax error: unexpected newline in composite literal; possibly missing comma or }
vendor/go.podman.io/common/pkg/config/new.go:59:34: syntax error: unexpected ) at end of statement
vendor/go.podman.io/common/pkg/config/new.go:62:3: syntax error: unexpected comma after top level declaration
make: *** [Makefile:383: bin/podman] Error 1
```

--- a/recipe/0002-relative-paths-for-system-config-and-binaries.patch
+++ b/recipe/0002-relative-paths-for-system-config-and-binaries.patch
@@ -54,7 +54,7 @@ index d40ff82..1a596a0 100644
  	return &paths{
 -		usr:  defaultContainersConfig,
 -		etc:  etcPath,
-+	        usr:  PathRelativeToExe(defaultContainersConfig)
++	        usr:  PathRelativeToExe(defaultContainersConfig),
 +		etc:  PathRelativeToExe(etcPath),
  		home: homePath,
  		uid:  unshare.GetRootlessUID(),

